test.cron

```diff
@@ -0,0 +1 @@
+* * * * * python3 /Users/collinschreyer/GSA/srt-fbo-scraper/src/fbo_scraper/main.py
```

Comment:
I would recommend changing this to something like:

```
* * * * * fbo_scraper
```

So anyone can use after pip installing the scraper.

since you should be able to run this inside the srt-fbo-scraper directory
```
pip install -e .[dev]
```
and you'll have the fbo_scraper command line option